src/ontology/cl-edit.owl

```diff
@@ -30820,6 +30834,119 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32555229") Annotation(ob
+AnnotationAssertion(rdfs:label obo:CL_4033016 "secondary crest myofibroblast")
+SubClassOf(obo:CL_4033016 obo:CL_0000186)
+SubClassOf(obo:CL_4033016 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002299))
+
+# Class: obo:CL_4033017 (bronchiolar smooth muscle cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "ISBN:9781260462982") Annotation(oboInOwl:hasDbXref "PMID:15347627") obo:IAO_0000115 obo:CL_4033017 "A smooth muscle cell that is part of a bronchiole.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033017 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033017 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033017 "2023-03-16T09:38:17Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033017 "bronchiolar smooth muscle cell")
+EquivalentClasses(obo:CL_4033017 ObjectIntersectionOf(obo:CL_0000192 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002186)))
+
+# Class: obo:CL_4033018 (lung megakaryocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33351116") obo:IAO_0000115 obo:CL_4033018 "A megakaryocyte that is resident in the lung connective tissue.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033018 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033018 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033018 "2023-03-16T11:31:23Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33351116") oboInOwl:hasRelatedSynonym obo:CL_4033018 "lung MKs")
+AnnotationAssertion(rdfs:label obo:CL_4033018 "lung megakaryocyte")
+SubClassOf(obo:CL_4033018 obo:CL_0000556)
+
+# Class: obo:CL_4033019 (serous secreting cell of trachea gland)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29656943") Annotation(oboInOwl:hasDbXref "PMID:31651362") obo:IAO_0000115 obo:CL_4033019 "A serous secreting cell that is part of a submucosal gland of the trachea.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033019 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033019 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033019 "2023-03-16T12:51:06Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4033019 "serous secreting cell of trachea gland")
+EquivalentClasses(obo:CL_4033019 ObjectIntersectionOf(obo:CL_0000313 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005203)))
+SubClassOf(obo:CL_4033019 obo:CL_0000313)
```

Comment:
Is this necessary given the equivalence axiom?

Reply:
No, thank you.